### PR TITLE
Remove Pull Request comment

### PR DESCRIPTION
**Pull request by omerzi**
- [x] All [tests](https://github.com/jfrog/froggit-go/actions/workflows/test.yml) passed. If this feature is not already covered by the tests, I added new tests.
- [x] I used `go fmt ./...` for formatting the code before submitting the pull request.
- [x] This feature is included on all supported VCS providers - GitHub, Bitbucket cloud, Bitbucket server, and GitLab.

---

### CODE REVIEW

File: vcsclient/github.go [hunk 1]
```diff
@@ -396,6 +396,23 @@ func (client *GitHubClient) ListPullRequestComments(ctx context.Context, owner,
...
+	if err != nil {
+		return err
+	}
+	ghClient, err := client.buildGithubClient(ctx)
+	if err != nil {
+		return err
+	}
+	resp, err := ghClient.Issues.DeleteComment(ctx, owner, repository, int64(commentID))
+	if err != nil || resp.StatusCode > 300 {
+		return fmt.Errorf("error occurred while deleting pull request comment:\n%s", err.Error())
```
Comment (sverdlov93) [inline vcsclient/github.go:411]: err can be nil

File: vcsclient/github.go [hunk 2]
```diff
@@ -396,6 +396,23 @@ func (client *GitHubClient) ListPullRequestComments(ctx context.Context, owner,
...
+	err := validateParametersNotBlank(map[string]string{"owner": owner, "repository": repository})
+	if err != nil {
+		return err
+	}
+	ghClient, err := client.buildGithubClient(ctx)
+	if err != nil {
+		return err
+	}
+	resp, err := ghClient.Issues.DeleteComment(ctx, owner, repository, int64(commentID))
+	if err != nil || resp.StatusCode > 300 {
```
Comment (sverdlov93) [inline vcsclient/github.go:410]: status code 300 is ok?

File: vcsclient/github.go (hunk 1 above)
Comment (omerzi) [reply vcsclient/github.go:411]: done

File: vcsclient/github.go (hunk 2 above)
Comment (omerzi) [reply vcsclient/github.go:410]: done

**Pull request merged**